### ISSUE TITLE
Pruned list of default outputs in CIME-EAMxx runs

### ISSUE THREAD

**Issue by AaronDonahue** [E3SM-Project/scream#1734]
Currently the default set of output for an EAMxx run may lead to unnecessarily big files with output variables we don't actually need.  So this issue is to start a discussion about which variables we want to keep in the default.  The current list, organized by process that changes it, is:
```
      # HOMME
      - horiz_winds
      - ps
      - pseudo_density
      - omega
      - p_int
      - p_mid
      # SHOC
      - cldfrac_liq
      - eddy_diff_mom
      - horiz_winds
      - sgs_buoy_flux
      - tke
      - inv_qc_relvar
      - pbl_height
      # CLD
      - cldfrac_ice
      - cldfrac_tot
      # SPA
      - aero_g_sw
      - aero_ssa_sw
      - aero_tau_lw
      - aero_tau_sw
      - nccn
      # P3
      - bm
      - nc
      - ni
      - nr
      - qi
      - qm
      - qr
      - T_prev_micro_step
      - qv_prev_micro_step
      - eff_radius_qc
      - eff_radius_qi
      - precip_ice_surf
      - precip_liq_surf
      # SHOC + P3
      - qc
      - qv
      # SHOC + P3 + RRTMGP + HOMME
      - T_mid
      # RRTMGP
      - sfc_alb_dif_nir
      - sfc_alb_dif_vis
      - sfc_alb_dir_nir
      - sfc_alb_dir_vis
      - LW_flux_dn
      - LW_flux_up
      - SW_flux_dn
      - SW_flux_dn_dir
      - SW_flux_up
      - rad_heating_pdel
      - sfc_flux_lw_dn
      - sfc_flux_sw_net
      # Diagnostics
      - PotentialTemperature
```

**Comment by AaronDonahue** [comment]
I think there is agreement we should remove the SPA section of variables since they are just regridded variables from the SPA data files anyway.

**Comment by AaronDonahue** [comment]
We probably don't need the T_prev_micro_step and qv_prev_micro_step either, those are basically P3 internal variables.

**Comment by PeterCaldwell** [comment]
I agree with both of Aaron's suggestions above. in addition, I'd remove
```
- sfc_alb_dif_nir
- sfc_alb_dif_vis
- sfc_alb_dir_nir
- SW_flux_dn_dir
```

**Comment by AaronDonahue** [comment]
@PeterCaldwell , but keep
```
      - sfc_alb_dir_vis
```

**Comment by PeterCaldwell** [comment]
> but keep sfc_alb_dir_vis

Yeah, we should have one albedo variable, but all 4 of them were more or less identical (at least for a first-glance sort of analysis).

**Comment by AaronDonahue** [comment]
how about `inv_qc_relvar` from SHOC?  And the `eff_radius_qc/qi` from P3?

**Comment by crterai** [comment]
I always copy and paste a default output yaml file for my purposes, so don't have as strong opinion, but `horiz_winds` is named twice. Does that produce two variables? I don't think so, but would be good to remove one of them. 
Also, I was also thinking `inv_qc_relvar` as potentially unnecessary since we don't use it in P3 anymore. And are the following necessary for a broad view of the climate? `rad_heating_pdel` `SW_flux_dn_dir`

**Comment by AaronDonahue** [comment]
Solved by #1735
